spec/reek/report/xml_report_spec.rb

```diff
@@ -27,8 +28,8 @@
     end
 
     it 'prints non-empty checkstyle xml' do
-      sample_path = File.expand_path 'checkstyle.xml', 'spec/samples'
-      expect { instance.show }.to output(File.read(sample_path)).to_stdout
+      sample_path = Pathname("#{__dir__}/../../samples/checkstyle.xml")
```

Comment:
Perhaps we can create a helper method in `spec_helper.rb` that gives us paths in the samples directory, so we don't have to mentally handle the `..`?